### ISSUE TITLE
Determine the goal of the project.

### ISSUE THREAD

**Issue by colshacol** [colshacol/thebab#3]
## Steps to reproduce:
1. Think about what we're going to do.

**Expected:** The thought comes easy.
**Actual:** It does not come easy.

## Issue details:
_Environment: development_

**Comment by colshacol** [comment]
We should figure this out.

**Comment by ClintonBaker** [comment]
Lets start with what it is we actually want to display. Think about how we would use it from a user's perspective and figure it out from there?

**Comment by colshacol** [comment]
Here are some links to the lists we want to replace:

https://github.com/enaqx/awesome-react
https://github.com/brillout/awesome-react-components
https://github.com/sorrycc/awesome-javascript
https://github.com/micromata/awesome-javascript-learning
https://github.com/vinta/awesome-python

The problems with these lists is that they are not updated very often and outdated shit doesn't get removed. Users can't discuss them or do anything dynamic to interact with each other in regards to the awesome things.

**Comment by ClintonBaker** [comment]
So I say we start by figuring out how to input the info to be displayed. Then build a simple display, and go from there.

**Comment by colshacol** [comment]
So we want to let users browse how?

My first thought is a hierarchy that somewhat reflects a NoSQL doc:
- Languages
  - Tool Types
    - Sub Categories
      - Specific Tools

For example:
- JavaScript
  - Browser Extensions
    - React Dev Tools
  - Open Source Projects
    - Babel
  - Editors
    - VS Code
  - Dev Tools
    - Webpack
  - Libraries
    - Lodash
  - Frameworks
    - React

**Comment by colshacol** [comment]
I feel that knowing _what_ we will display will help us make better decisions on how to display it.

**Comment by ClintonBaker** [comment]
That's fair. So figure out what we actually need to display, build the input, then build the display for said input.

**Comment by colshacol** [comment]
Yeppers. And somewhere in there we need to build the output, haha. So when there is input, we can display the output.

Maybe that is what you meant by "build the display for said input." but I thought you meant the components for allowing things to be added.

**Comment by colshacol** [comment]
Or maybe we should shy away from the `<ul>` structure, and just cultivate an ecosystem of tags that connect everything and allow us to better study and interpret the data we generate.

**Comment by ClintonBaker** [comment]
Well, we need some sort of easily navigable structure...

**Comment by ClintonBaker** [comment]
Could do a nav bar on the left if you're into that sort of thing

**Comment by colshacol** [comment]
I am. Not a bad idea. It is a bit difficult to work with though. One problem I have always had with it:

If somebody is on a big or wide screen, how do you position shit?

Your nav is all to the left like 10 inches from your content. And your content cant really stretch 100% or it gets all kind of horizontally stretched feeling. Way too wide.

**Comment by colshacol** [comment]
Well, since we wanna get a move on, let's go ahead and just solidify what we know we need for the app feature of awesome-things display.

**Comment by ClintonBaker** [comment]
Media queries? On ridiculously wide monitors you can keep the app more toward the center. Like this | ||     | |
                ^ Nav goes here

**Comment by colshacol** [comment]
Yeah, but then you have a 2003 style nav floating there from top to bottom and 6 inches from the left!

Let's look at some examples of side navs and see how it is handled.

**Comment by ClintonBaker** [comment]
That's true! A top level nav could work if the width ends up being too much of an issue
